src/iocore/hostdb/P_RefCountCache.h

```diff
@@ -428,21 +428,19 @@ RefCountCache<C>::RefCountCache(unsigned int num_partitions, int size, int items
                                 const std::string metrics_prefix)
   : header(RefCountCacheHeader(object_version))
 {
-  ts::Metrics &intm = ts::Metrics::getInstance();
-
   this->max_size       = size;
   this->max_items      = items;
   this->num_partitions = num_partitions;
 
-  this->rsb.refcountcache_current_items        = intm.newMetricPtr((metrics_prefix + "current_items").c_str());
-  this->rsb.refcountcache_current_size         = intm.newMetricPtr((metrics_prefix + "current_size").c_str());
-  this->rsb.refcountcache_total_inserts        = intm.newMetricPtr((metrics_prefix + "total_inserts").c_str());
-  this->rsb.refcountcache_total_failed_inserts = intm.newMetricPtr((metrics_prefix + "total_failed_inserts").c_str());
-  this->rsb.refcountcache_total_lookups        = intm.newMetricPtr((metrics_prefix + "total_lookups").c_str());
-  this->rsb.refcountcache_total_hits           = intm.newMetricPtr((metrics_prefix + "total_hits").c_str());
-  this->rsb.refcountcache_last_sync_time       = intm.newMetricPtr((metrics_prefix + "last_sync.time").c_str());
-  this->rsb.refcountcache_last_total_items     = intm.newMetricPtr((metrics_prefix + "last_sync.total_items").c_str());
-  this->rsb.refcountcache_last_total_size      = intm.newMetricPtr((metrics_prefix + "last_sync.total_size").c_str());
+  this->rsb.refcountcache_current_items        = Metrics::Gauge::createPtr((metrics_prefix + "current_items").c_str());
```

Comment:
Why are you doing string -> c string -> string_view instead of just string -> string_view directly?

Reply:
Probably a remnant from the code as it was prior to changing to new metrics.